ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -2427,6 +2427,26 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-disasterrecovery-sanity
     workflow: cucushift-installer-rehearse-nutanix-ipi-proxy
+- as: nutanix-ipi-multi-disks-f28
+  cluster: build01
+  cron: 5 7 15 * *
+  steps:
+    cluster_profile: nutanix-qe-zone
```

Comment:
only zonal cluster profile support the feature ?

Reply:
Nutanix-qe also support, I'd like to run more jobs on nutanix-qe-zone in purpose

Reply:
I am thinking if we can merge the mult-disk feature to some existing install profile, e.g: `nutanix-ipi-image-registry-s3` or `nutanix-ipi-zones`, you know, we are adding more and more install profiles.

Reply:
nutanix-ipi-zones support both zonal and not zonal job, it's the first not zonal job on it plus it's an STS nutanix env, maybe we can merge other features to it in future

Reply:
Yeah, I understand `nutanix-qe-zone` can support both zonal and not zonal job, the multi-disks feature should not affect zonal testing much, to consider that, once any new install profile is introduced, it will also be covered in upgrade ci (including y-stream upgrade and z-stream upgrade), so I think this is a good chance to merge it with `nutanix-ipi-zones`.

Reply:
support multi-disks in nutanix-ipi-zones is another case, I'll check if multi-disks works in another PR

Reply:
If that, now we have 4.18 ci enabled, so also help add 4.17 z-stream upgrade job and 4.18 ci job and 4.18 z-stream and y-stream job.

Reply:
Got it, if that's the case, merge multi-disks to `nutanix-ipi-image-registry-s3`, should I also change job name like `nutanix-ipi-image-registry-s3-data-disks-f28` accordingly in all 4.17 z-stream upgrade job and 4.18 ci job and 4.18 z-stream and y-stream job?

Reply:
Right, the same rule.

Reply:
@jianlinliu both GPU and multi-disks are not added to 4.17 z-stream upgrade job and 4.18 ci job and 4.18 y-stream upgrade ci job from 4.17, so I'm planning to add both of them, is it OK?

Reply:
okay for me.